app/controllers/users_controller.rb

```diff
@@ -1,14 +1,16 @@
-class UsersController < Clearance::UsersController
+class UsersController < ApplicationController
+  before_action :redirect_signed_in_users
+
   def new
-    @user = user_from_params
+    @user = User.new
```

Comment:
any reason for this not to be `User.new(user_params)` ?

Reply:
What's the benefit in here? `Resource.new` is default in Rails. Params are not present in here, since form wasn't submitted yet. If used, it will make possible to fill in form using query params. Is that something we would like to have?

Reply:
just because it was `user_from_params` previously

Reply:
done